### PR TITLE
Update KG content and add new publications

### PR DESCRIPTION
**Pull request by YojanaGadiya**

### CODE REVIEW

File: content/recipes/introduction/public-kg-resource-integration.md [hunk 1]
```diff
@@ -274,34 +254,28 @@ several ontology "slims", which are simplified versions of larger semantic artef
...
+OpenBioLink is an open benchmark dataset that serve as training or evaluation datasets for link prediction models {footcite}`rpath_paper_ddomingo`, {footcite}`drug2ways_ddomingo`. It is an ensembl of large independent resource connecting drugs to disease via proteins. These resources include STRING, GO, DisGeNet, HPO, STITCH, CTD, HPO, DrugCentral, and SIDER. 
 
+Taking a deeper dive into the KG statistics, there are 24,806 entities that are either drugs, proteins, indications or phenotype and 2,35.881 triples that are part of 4 edge-types.
 
-<!--
-## OpenBiolink
+GitHb Repository: https://openbiolink.github.io/
 
-https://openbiolink.github.io/
-https://openbiolink.github.io/dataset/
```
Comment (YojanaGadiya) [inline content/recipes/introduction/public-kg-resource-integration.md:279]: @proccaserra can you write 1-2 sentences here explaining what the code does?

File: content/recipes/introduction/public-kg-resource-integration.md [hunk 2]
```diff
@@ -311,16 +285,14 @@ dl = OBL2021Dataset()
 train = dl.training # torch.tensor of shape(num_train,3)
 valid = dl.validation # torch.tensor of shape(num_val,3)
 ```
```
Comment (YojanaGadiya) [inline content/recipes/introduction/public-kg-resource-integration.md:287]: @proccaserra can you write 1-2 sentences here explaining what the code does?

File: content/recipes/introduction/public-kg-resource-integration.md [hunk 3]
```diff
@@ -409,54 +364,43 @@ result = pipeline(
...
-and make novel link predictions:
-
-https://github.com/tomasonjo/blogs/blob/master/pykeen/Hetionet%20-%20RotatE.ipynb
-
-
+To see an example of how to use the library, we highly recommended reading the [Medium blog post](https://towardsdatascience.com/knowledge-graph-completion-with-pykeen-and-neo4j-6bca734edf43) and consult the associated jupyter notebook, which shows how to use your own dataset, understand the evaluation, and make novel link predictions:
 
+Link Prediction example notebook: https://github.com/tomasonjo/blogs/blob/master/pykeen/Hetionet%20-%20RotatE.ipynb
 
 ---
```
Comment (FuqiX) [inline content/recipes/introduction/public-kg-resource-integration.md:371]: is PDBe-KG https://www.ebi.ac.uk/pdbe/pdbe-kb/graph-download also in the scope of this article?

File: content/recipes/introduction/public-kg-resource-integration.md [hunk 4]
```diff
@@ -27,44 +27,37 @@ In the following sections, we will cover the following topics:
 
 ## Introduction
 
-When it comes to research carried out by academic and industry ventures such as the Innovative Medicine Initiative of these
-European Union, three main scenarios can be outlined:
+When it comes to research carried out by academic and industry ventures such as the Innovative Medicine Initiative of these European Union, three main scenarios can be outlined:
```
[the remark this answers is not in the document]
Comment (proccaserra) [reply content/recipes/introduction/public-kg-resource-integration.md:30]: you are right @FuqiX, even though repurposing could be seen as a subtype of 1. identification of new drug targets.

File: content/recipes/introduction/public-kg-resource-integration.md [hunk 5]
```diff
@@ -27,44 +27,37 @@ In the following sections, we will cover the following topics:
...
 
-Yummydata is a catalog of SPARQL endpoints of interest to the biomedical community, 
-maintained by the Database Center for Life Sciences in Kashiwa, Japan {footcite}`pmid29688370`. 
+Yummydata is a catalog of SPARQL endpoints of interest to the biomedical community, maintained by the Database Center for Life Sciences in Kashiwa, Japan {footcite}`pmid29688370`. 
 
 Database URL: http://yummydata.org/
 
-The resource can be viewed as a registry to resources as well as a monitoring services, which regularly checks the "health status"
-of the resources it indexes.
+The resource can be viewed as a registry to resources as well as a monitoring services, which regularly checks the "health status" of the resources it indexes.
```
[the remark this answers is not in the document]
Comment (proccaserra) [reply content/recipes/introduction/public-kg-resource-integration.md:52]: +1

File: content/recipes/introduction/public-kg-resource-integration.md (hunk 2 above)
Comment (proccaserra) [reply content/recipes/introduction/public-kg-resource-integration.md:287]: will do

**Review by proccaserra — APPROVED** [review]
thx @YojanaGadiya @FuqiX , I'll add according to your input

**Pull request merged**